app/services/payment_provider_customers/gocardless_service.rb

```diff
@@ -77,5 +88,38 @@ def deliver_error_webhook(gocardless_error)
         },
       )
     end
+
+    def create_billing_request(gocardless_customer_id)
+      client.billing_requests.create(
+        params: {
+          mandate_request: {
+            scheme: 'bacs',
+          },
+          links: {
+            customer: gocardless_customer_id,
+          }
+        }
+      )
+    rescue GoCardlessPro::Error => e
+      deliver_error_webhook(e)
+
+      raise
+    end
+
+    def create_billing_request_flow(billing_request_id)
+      client.billing_request_flows.create(
+        params: {
+          redirect_uri: 'https://gocardless.com/',
+          exit_uri: 'https://gocardless.com/',
+          links: {
+            billing_request: billing_request_id,
+          }
+        }
```

Comment:
Style/TrailingCommaInArguments: Put a comma after the last parameter of a multiline method call. (https://rubystyle.guide#no-trailing-params-comma)